{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "0ed39735_b4330327",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 635
      },
      "writtenOn": "2022-03-11T16:14:08Z",
      "side": 1,
      "message": "missing python3-oslo-concurrency and python3-oslo-log",
      "revId": "f6787269854a9323712e524aa48e4e15a0129473",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    }
  ]
}